### PR TITLE
Bump version to v4.9

### PR DESCRIPTION
**Pull request by umohnani8**

### CODE REVIEW

**Comment by umohnani8** [comment]
@rhatdan @lsm5 PTAL

File: setup.cfg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = podman
-version = 4.9.0-dev
+version = 4.9.0
 author = Brent Baude, Jhon Honce
 author_email = jhonce@redhat.com
```
Comment (lsm5) [inline setup.cfg:5]: @umohnani8 maybe you should add / update with your name here. wdyt @jwhonce ?

**Review by lsm5 — APPROVED** [review]
/lgtm
/hold

File: setup.cfg (hunk 1 above)
Comment (umohnani8) [reply setup.cfg:5]: I will do it in the next update :)

**Pull request merged**